src/webloader.js

```diff
@@ -108,15 +108,41 @@ function checkInstanceCount(evolv) {
 	}
 }
 
+function setUid(uid) {
+	if (!uid) {
+		return;
+	}
+
+	const script = currentScript();
+	script.dataset.evolvUid = uid;
+	main();
+}
+
+function checkLazyUid(script) {
+	if ((script.dataset.evolvLazyUid && !script.dataset.evolvUid) || ("evolvUid" in script.dataset && !script.dataset.evolvUid)) {
+		script.dataset.evolvLazyUid || console.warn('Evolv uid is empty - experiment will not run until evolv.setUid() is called. Please use data-evolv-lazy-uid="true" when setting a lazy uid.');
```

Comment:
Nitpick: not crazy about this syntax:
```suggestion
		if (!script.dataset.evolvLazyUid) {
		  console.warn('Evolv uid is empty - experiment will not run until evolv.setUid() is called. Please use data-evolv-lazy-uid="true" when setting a lazy uid.');
		}
```